java/omniscijdbc/src/main/java/com/omnisci/jdbc/OmniSciPreparedStatement.java

```diff
@@ -346,17 +351,41 @@ public void addBatch() throws SQLException {
     if (isInsert) {
       // take the values and use stream inserter to add them
       if (isNewBatch) {
+        // check for columns names 
+        Matcher matcher = REGEX_LOF_PATTERN.matcher(currentSQL);
+        if (matcher.find()) {
+          String listOfFieds[] = matcher.group(1).trim().split("\\s*,+\\s*,*\\s*");
```

Comment:
Can you switch this to say `listOfFields` (i.e. fix the misspelling of `Fieds`) ?

Reply:
did a commit to correct the misspelling